test/e2e/utils/test_context.go

```diff
@@ -232,9 +232,12 @@ func (t *TestContext) Tidy() error {
 // Destroy is for cleaning up the docker images for testing
 func (t *TestContext) Destroy() {
 	//nolint:gosec
-	cmd := exec.Command("docker", "rmi", "-f", t.ImageName)
-	if _, err := t.Run(cmd); err != nil {
-		warnError(err)
+	// if image name is not present or not provided skip execution of docker command
+	if t.ImageName != "" {
```

Comment:
We could either use trim here to avoid the edge case " " string.Trim(t.ImageName) and then you check the len to see if is < 0. 

However, it is a nit I am fine to move forward with as it is .